.gitignore

```diff
@@ -17,3 +17,8 @@ doc_src/auto_api/
 doc_src/auto_examples/
 auto_figures/
 enhancement.png
+examples/data_files.csv
+examples/single_mutant_overlay.pdf
+pyspecProcScripts.egg-info/
+pyspecProcScripts/__init__.py.orig
```

Comment:
just delete this file and remove from here (the .orig are created when resolving conflicts)